### DISCUSSION TITLE
Riptide hot icon not showing

### DISCUSSION THREAD

**Discussion by Puddux** [rgd87/Aptechka#101] [General]
Hi there,

maybe it's a known issue, or maybe it's just my configuration, but I can't manage to show the HoT icon on the raidframes after casting Riptide on a target. It doesn't show neither the icon nor the bars (as per default configuration).

With other HoTs the addon works correctly (e.g. Renew), but Riptide looks different.

Thanks for your help

**Comment by d87** [comment]
Should be for some time now, Riptide had incorrect ID
